shared/src/test/scala/org/adridadou/openlaw/vm/OpenlawVmSpec.scala

```diff
@@ -87,6 +87,40 @@ class OpenlawVmSpec extends FlatSpec with Matchers {
     vm1.signature(identity2.email) shouldBe None
   }
 
+  it should "be possible to sign the contract with an external signature" in {
+    val template = """this is a contract [[Signatory:ExternalSignature(service:"DocuSign")]]"""
```

Comment:
Shouldn't we pass the `identity` argument to `ExternalSignature`?
Something like
```
[[identity: Identity]]
this is a contract [[Signatory:ExternalSignature(service:"DocuSign", identity: identity)]]
```